README.md

```diff
@@ -1,113 +1,13 @@
-
-## Death Dependencies
-
-With kubexit, you can define death dependencies between processes that are wrapped with kubexit and configured with the same graveyard.
-
-If the dependency process (ex: a stateless webapp) exits before the dependent process (ex: a sidecar proxy), kubexit will detect the tombstone update (`Died: <timestamp>`) and send the `TERM` signal to the dependent process.
-
-The primary use case for this feature is Kubernetes Jobs, where a sidecar container needs to be gracefully shutdown when the primary container exits, otherwise the Job will never complete.
-
-## Config
-
-kubexit is configured with environment variables only, to make it easy to configure in Kubernetes and minimize entrypoint/command changes.
-
-General:
-- `KUBEXIT_LOG_LEVEL` - The log level for logged messages. Default: `info`.
-
-Tombstone:
-- `KUBEXIT_NAME` - The name of the tombstone file to use. Must match the name of the Kubernetes pod container, if using birth dependency.
-- `KUBEXIT_GRAVEYARD` - The file path of the graveyard directory, where tombstones will be read and written.
-
-Death Dependency:
-- `KUBEXIT_DEATH_DEPS` - The name(s) of this process death dependencies, comma separated.
-- `KUBEXIT_GRACE_PERIOD` - Duration to wait for this process to exit after a graceful termination, before being killed. Default: `30s`.
-
-Birth Dependency:
-- `KUBEXIT_BIRTH_DEPS` - The name(s) of this process birth dependencies, comma separated.
-- `KUBEXIT_BIRTH_TIMEOUT` - Duration to wait for all birth dependencies to be ready. Default: `30s`.
-- `KUBEXIT_POD_NAME` - The name of the Kubernetes pod that this process and all its siblings are in.
-- `KUBEXIT_NAMESPACE` - The name of the Kubernetes namespace that this pod is in.
-
-## Install
-
-While kubexit can easily be installed on your local machine, the primary use cases require execution within Kubernetes pod containers. So the recommended method of installation is to either side-load kubexit using a shared volume and an init container, or build kubexit into your own container images.
-
-Build from source:
-
-```
-go get github.com/karlkfi/kubexit/cmd/kubexit
-```
-
-Copy from pre-built Alpine-based container image in a multi-stage build:
-
-```
-FROM karlkfi/kubexit:latest AS kubexit
-
-FROM alpine:3.11
-RUN apk --no-cache add ca-certificates tzdata
-COPY --from=kubexit /bin/kubexit /bin/
-ENTRYPOINT ["kubexit"]
-```
-
-Copy from init container to ephemeral volume:
-
-```
-volumes:
-- name: kubexit
-  emptyDir: {}
-
-initContainers:
-- name: kubexit
-  image: karlkfi/kubexit:latest
-  command: ['cp', '/bin/kubexit', '/kubexit/kubexit']
-  volumeMounts:
-  - mountPath: /kubexit
-    name: kubexit
-```
-
-## Examples
-
-- [Client Server Job](examples/client-server-job/)
-- [CloudSQL Proxy Job](examples/cloudsql-proxy-job/)
+docker run --name fn-kubexit
+```
```

Comment:
If applicable, please add instructions to update the dependencies/logic.

Reply:
Yeah, add instructions to:
- add more patches
- update the base version

Reply:
Updated https://github.com/acquia/kubexit/blob/d5e2c57fc89fae93f34efeafa2096c0ee3c70789/README.md